lib/parameter/parametrizable.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Parameter.Parametrizable do
```

Comment:
I think can still use type module for the behaviour of implementing custom types

Reply:
actually this approach is more cleaner, let's do this :)